src/components/OldApp/OldApp.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import WebcamCaptureContainer from '../WebcamCapture/WebcamCaptureContainer.js';
+import WebcamCaptureContainer from '../WebcamCapture/container';
```

Comment:
How come this file name has reverted back?

Reply:
My guess? witchcraft!